src/utils.py

```diff
@@ -20,23 +23,43 @@ def __init__(self, data_path):
         download_logger.info(f"Data Handler Class Created - {data_path}")
         self.data_path.mkdir(parents=True, exist_ok=True)
 
-    def downloader(self, download_type):
+    def downloader(self, download_file_name):
         """
-        Downloads download_type in data_path/download_type.
+        Downloads download_file_name in data_path/download_file_name.
         """
-        download_logger.info(f"Downloading {download_type}")
-        fname = f"{download_type}.tar.gz"
+        download_logger.info(f"Downloading {download_file_name}")
+        fname = f"{download_file_name}.tar.gz"
         if not (self.data_path / fname).exists():
             google_drive_link = {
                 "indosum": "https://drive.google.com/uc?export=download&id=1OgYbPfXFAv3TbwP1Qcwt_CC9cVWSJaco",
-                "indo_lm": "https://drive.google.com/uc?export=download&id=14uhX9s43eKAsy7b94FV5mHn0vDeJc4YH",
+                "indo_lm": "https://drive.google.com/uc?export=download&id=1p9JSui5R2aRHLCh_gKgD1X0H5b3sVrsE",
+                "sentencepiece": "https://drive.google.com/uc?export=download&id=1bWmSiHhp6i8xsA6AZuL4sGjzbFfNCTyt",
             }
-            url = google_drive_link[download_type]
+            url = google_drive_link[download_file_name]
             output = f"{self.data_path/fname}"
             gdown.download(url, output, quiet=False)
-        if not (self.data_path / f"{download_type}").exists():
+        if not (self.data_path / f"{download_file_name}").exists():
             output_unzip = subprocess.check_output(
                 ["tar", "xvzf", f"{self.data_path/fname}", "-C", f"{self.data_path}"]
             )
             download_logger.info(f"{output_unzip}")
-        download_logger.info(f"{download_type} Downloaded")
+        download_logger.info(f"{download_file_name} Downloaded")
+
+
+class LangTokenizer(BaseTokenizer):
```

Comment:
`LangTokenizer` class should be imported and used as is from fastai. It already has a lang param IIRC.

Reply:
Using LangTokenizer while loading databunch